src/main/scala/org/dia/algorithms/mcs/MCSUtils.scala

```diff
@@ -118,11 +123,13 @@ object MCSUtils {
       datafile.close()
     }
     catch {
-      case _: Throwable => logger.info("Error generating netCDF file for " + filepath + "\n")
+      case _: Throwable => logger.info("Error generating netCDF file for " + currFile + "\n")
```

Comment:
If i remember correctly the try catch is only needed for the datafile.close() statement.
If that's the case can we limit the try block to just datafile.close or just the lines of code that throw the error?

I don't think we need to capture the lat/lon variable initialization step in the try block.